Stable-Branch-Strategy.md

```diff
@@ -80,19 +80,23 @@ stable and master. While this is not in place currently, it should be considered
 
 ### Patch releases
 
-Releases are made on a weekly cadence for patch releases, which include a GitHub release as
+Releases are normally made every-other-week for patch releases, which include a GitHub release as
```

Comment:
Nit: I'd remove the hypens in deference to non native speakers.

Reply:
Good call.